command/tls/ca/create/tls_ca_create.go

```diff
@@ -32,13 +33,12 @@ type cmd struct {
 func (c *cmd) init() {
 	c.flags = flag.NewFlagSet("", flag.ContinueOnError)
 	// TODO: perhaps add a -years arg to better capture user intent given that leap years are a thing
-	c.flags.IntVar(&c.days, "days", 1825, "Provide number of days the CA is valid for from now on. Defaults to 5 years.")
-	c.flags.BoolVar(&c.constraint, "name-constraint", false, "Add name constraints for the CA. Results in rejecting "+
-		"certificates for other DNS than specified. If turned on localhost and -domain will be added to the allowed "+
-		"DNS. If the UI is going to be served over HTTPS its DNS has to be added with -additional-constraint. It is not "+
-		"possible to add that after the fact! Defaults to false.")
-	c.flags.StringVar(&c.domain, "domain", "consul", "Domain of consul cluster. Only used in combination with -name-constraint. Defaults to consul.")
-	c.flags.StringVar(&c.clusterID, "cluster-id", "", "ClusterID of the consul cluster, requires -domain to be set as well. When used will set URIs with spiffeid.")
+	c.flags.IntVar(&c.days, "days", 1825, "Provide number of days for which the CA will be valid. Defaults to 5 years.")
```

Comment:
I know that this is existing but the word "Provide" here seems redundant to me since the purpose of flags is to allow users to provide arguments.

```suggestion
	c.flags.IntVar(&c.days, "days", 1825, "Number of days the CA is valid for. Defaults to 1825 days (approximately 5 years).")
```